lambda_functions/ml_service/template.yaml

```diff
@@ -51,6 +51,7 @@ Resources:
         - arn:aws:iam::aws:policy/ComprehendFullAccess
         - arn:aws:iam::aws:policy/AmazonS3FullAccess
         - arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
+        - arn:aws:iam::aws:policy/APIGatewayFullAccess 
```

Comment:
Idealy we would create our own policy that allows only access on the resources that this service actually needs to access. But this is not a must have in my opinion.

Reply:
Yes, I also favour to restrict access as much as possible. That is meant as first step to get things running.